### PR TITLE
Fix missing photos var and add photo title

### LINKED ISSUE #1354: Notifications emails not sending
### Detailed description of the problem [REQUIRED]
Notifications emails are not sending for new photos.  
Error in log:
`2022-05-29 07:00:07 UTC -- error   -- ::require -- 13 -- Undefined variable $photos
2022-05-29 07:00:07 UTC -- error   -- App\Console\Commands\PhotosAddedNotification::handle -- 90 -- Undefined variable $photos; caused by
2022-05-29 07:00:07 UTC -- error   -- Livewire\LivewireViewCompilerEngine::handleViewException -- 13 -- Undefined variable $photos (View: /mnt/server/Lychee/resources/views/emails/photos-added.blade.php); caused by
2022-05-29 07:00:07 UTC -- error   -- App\Console\Commands\PhotosAddedNotification::handle -- 90 -- Undefined variable $photos (View: /mnt/server/Lychee/resources/views/emails/photos-added.blade.php); caused by`

### Steps to reproduce the issue
**Steps to reproduce the behavior:**
1. Add new photos
2. Wait week for email job to trigger
3. check logs
4. See error*

### Output of the diagnostics  [REQUIRED]

    Diagnostics
    -----------
    Warning: Dropbox import not working. dropbox_key is empty.
    Warning: zend.assertions is disabled although Lychee is in debug mode. For easier debugging code generation for assertions should be enabled.

    System Information
    ------------------
    Lychee Version (git):            master (9174540) --  - 1 commits behind master 690ec0e (0 seconds ago)
    DB Version:                      4.5.1
    
    composer install:                dev
    APP_ENV:                         production
    APP_DEBUG:                       true
    
    System:                          Linux
    PHP Version:                     8.1
    PHP User agent:                  Lychee/4 (https://lycheeorg.github.io/)
    Timezone:                        America/Los_Angeles
    Max uploaded file size:          500M
    Max post size:                   500M
    Max execution time:              200
    MySQL Version:                   10.6.7-MariaDB-2ubuntu1
    
    Imagick:                         1
    Imagick Active:                  1
    Imagick Version:                 1691
    GD Version:                      2.3.0

    Config Information
    ------------------
    version:                         040501
    check_for_updates:               1
    sorting_photos_col:              created_at
    sorting_photos_order:            DESC
    sorting_albums_col:              max_taken_at
    sorting_albums_order:            DESC
    imagick:                         1
    skip_duplicates:                 1
    small_max_width:                 0
    small_max_height:                360
    medium_max_width:                1920
    medium_max_height:               1080
    lang:                            en
    layout:                          1
    image_overlay_type:              none
    default_license:                 CC-BY-NC-ND-4.0
    compression_quality:             90
    full_photo:                      1
    delete_imported:                 1
    Mod_Frame:                       1
    Mod_Frame_refresh:               30
    thumb_2x:                        1
    small_2x:                        1
    medium_2x:                       1
    landing_page_enable:             1
    landing_owner:                   Vincent Juliano
    landing_title:                   Life on 35mm
    landing_subtitle:                Click top right to continue.  Next page top left to login
    landing_facebook:                
    landing_flickr:                  
    landing_twitter:                 
    landing_instagram:               
    landing_youtube:                 
    landing_background:              /uploads/medium/3d4bf4e35e5ea06853d7983c88efb6a2.jpg
    site_title:                      Vince Juliano's Photo Gallery
    site_copyright_enable:           1
    site_copyright_begin:            2021
    site_copyright_end:              2022
    additional_footer_text:          "Alex's Photos" copyright Alex Panicacci 2022
    display_social_in_gallery:       0
    public_search:                   0
    SL_enable:                       0
    SL_for_admin:                    0
    public_recent:                   0
    recent_age:                      60
    public_starred:                  0
    downloadable:                    1
    photos_wraparound:               1
    map_display:                     1
    zip64:                           1
    map_display_public:              1
    map_provider:                    Wikimedia
    force_32bit_ids:                 0
    map_include_subalbums:           1
    update_check_every_days:         3
    has_exiftool:                    1
    share_button_visible:            1
    import_via_symlink:              0
    has_ffmpeg:                      1
    location_decoding:               1
    location_decoding_timeout:       30
    location_show:                   1
    location_show_public:            1
    rss_enable:                      0
    rss_recent_days:                 7
    rss_max_items:                   100
    prefer_available_xmp_metadata:   0
    editor_enabled:                  1
    lossless_optimization:           1
    swipe_tolerance_x:               150
    swipe_tolerance_y:               250
    local_takestamp_video_formats:   .avi|.mov
    log_max_num_line:                10000
    unlock_password_photos_with_url_param: 0
    nsfw_visible:                    0
    nsfw_blur:                       0
    nsfw_warning:                    0
    nsfw_warning_admin:              0
    map_display_direction:           1
    album_subtitle_type:             oldstyle
    upload_processing_limit:         4
    public_photos_hidden:            1
    new_photos_notification:         1
    legacy_id_redirection:           1

### Browser and system
Ubuntu 22.04

### PR DESCRIPTION
**Pull request by qwerty287**
Just add the variable `photos` to fix https://github.com/LycheeOrg/Lychee/issues/1354. Also adds a var `title` with the photo's title set a alt text for the image.

### CODE REVIEW

File: resources/views/view.blade.php
Comment (nagmat84) [inline resources/views/view.blade.php:9]:
```suggestion
    <link rel="apple-touch-icon" href="{{ asset('img/apple-touch-icon-iphone.png') }}" sizes="120x120">
    <link rel="apple-touch-icon" href="{{ asset('img/apple-touch-icon-ipad.png') }}" sizes="152x152">
```

I have no experiences with Blade templates and don't fully understand how they work and how what final relative location looks like seen from the perspective of the client, but I assume that we need `asset` here as well to be fully on the save side? Unfortunately, I have no real chance to test it.

**Review by nagmat84 — APPROVED** [review]
LGTM, but untested because I have no mail server configured. I have a minor suggestion, but I am not sure about that.

File: resources/views/view.blade.php
Comment (qwerty287) [reply resources/views/view.blade.php:9]: No, because the files are exposed as they are in the `public` directory. This should be just how it is in the template in the final HTML.

Comment (nagmat84) [reply resources/views/view.blade.php:9]: OK, but in this case we should not use `asset` for `$photo['thumb']` in

https://github.com/LycheeOrg/Lychee/blob/fba9dcba7b350914fbf843004b1323ff5d62cbcf/resources/views/emails/photos-added.blade.php#L20

either. The URL is also relative to the `public` directory. The docs are not very clear about what `asset` does besides converting the given path into an absolute URL, but at least we should be consistent about its usage.

**Comment by nagmat84** [comment]
IMHO, it can be merged.

**Comment by qwerty287** [comment]
Then feel free to do it the next time, you don't have to wait for me :)

**Comment by nagmat84** [comment]
I was not sure if you wanted to wait for a second review. :)

**Pull request merged**
